.travis.yml

```diff
@@ -149,6 +151,10 @@ jobs:
       after_failure: post_message "TRAVIS_MESSAGE" "'make dist' failed"
 
 
+    - name: Re-check Shell Scriopts (after build) with shellcheck
+      script:
+        - shellcheck --format=gcc $(find . -name '*.sh' -not -iwholename '*.git*')
```

Comment:
Travis doesn't share state between stages, so this will just do the same thing as line 104.

Reply:
That wasn't the case being covered here; it was the case where template shell scripts go from `*.sh.in` to the end result `*.sh` and are re-checked in case the templating broke them in some way :)

Reply:
That still doesn't work though, because each individual job operates on a clean, completely fresh clone of the source repo. So to do this checking, you'd have to run the build in this job and then run the shellcheck command.

Reply:
Old; took a different route :)